terraform/eks-cluster/commands/cycle.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+set -ex
+
+# Usage ./commands/cycle.sh --all
+
+# Spin up app/cluster and run a build
+./commands/create.sh $@
```

Comment:
nit: args need quoting in case they have spaces: `"$@"`

(also super recommend installing the "shellcheck" and "Bash IDE" extensions for VSCode, they are really lightweight and make bash development more sane)

Reply:
Done